Lib/fontTools/designspaceLib/__init__.py

```diff
@@ -702,7 +718,7 @@ def readRules(self):
                 ruleObject.conditions.append(cd)
             for subElement in ruleElement.findall('.sub'):
                 a = subElement.attrib['name']
-                b = subElement.attrib['with']
+                b = subElement.attrib['byname']
```

Comment:
Where did the `with` come from - it used to be `byname`?

Reply:
no idea.. I found it there